test/runtests.jl

```diff
@@ -339,9 +339,9 @@ end
     # Test output
     @test sprint(show, ftest(mod, nullmod)) == 
         """
-                Res. DOF DOF    ΔDOF    SSR    ΔSSR    R²      ΔR²    F*       p(>F) 
-        Model 1 10.0000  3.0000         0.1283         0.9603                        
-        Model 2 11.0000  2.0000 -1.0000 3.2292 -3.1008 -0.0000 0.9603 241.6234 <1e-7 
+                Res. DOF DOF ΔDOF SSR    ΔSSR    R²      ΔR²    F*       p(>F) 
+        Model 1 10       3        0.1283         0.9603                        
```

Comment:
Can you align integers on the right? That makes it easier to read by columns since digits of the same rank are aligned. (BTW, if you do the same for floats, they will be automatically aligned on the decimal separator since their are fixed to four digits after it.)